src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/Cas3SoftDeleteApplicationTest.kt

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.cas3
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.givens.givenAUser
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.givens.givenAnOffender
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.UserEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.UserRole
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class Cas3SoftDeleteApplicationTest : IntegrationTestBase() {
+
+  @Test
+  fun `soft delete application without JWT returns 401`() {
+    webTestClient.delete()
+      .uri("/cas3/applications/${UUID.randomUUID()}")
+      .exchange()
+      .expectStatus()
+      .isUnauthorized
+  }
+
+  @Test
+  fun `soft delete application with incorrect ID returns 404`() {
```

Comment:
given you have unit tested this, i don't think it needs testing here as well (I appreciate existing integration tests may be doing this, i think we need to work on reducing those)